bin/install-ubuntu

```diff
@@ -13,4 +13,6 @@ apt-get update
 DEBIAN_FRONTEND=noninteractive apt-get install --yes python-pip python-dev libxml2-dev libxslt-dev libffi-dev
 apt-get remove --yes ansible || echo "ok"
 
-pip install -r -U $BINDIR/../requirements.txt
+pip install -r requirements.txt
```

Comment:
Seems like this would fail if somebody didn't follow directions and cd'd to bin/ before running ./install-ubuntu. Maybe we don't care, maybe we do?